python/pyiceberg/catalog/hive.py

```diff
@@ -548,10 +511,9 @@ def update_namespace_properties(
                 for key, value in updates.items():
                     parameters[key] = value
                     updated.add(key)
+
             open_client.alter_database(database_name, _annotate_namespace(database, parameters))
 
         expected_to_change = (removals or set()).difference(removed)
 
-        return PropertiesUpdateSummary(
-            removed=list(removed or []), updated=list(updates.keys() if updates else []), missing=list(expected_to_change)
-        )
+        return PropertiesUpdateSummary(removed=list(removed or []), updated=list(updated or []), missing=list(expected_to_change))
```

Comment:
I see there are multiple places with this kind of style change and newline additions. If not required by linter, can we avoid these changes?

Reply:
This is done my `make lint`. I changed it to the previous formatting. `make lint` put it back in one line. 
I didn't find another example like this. Please point them out and I'll fix it.